src/libs/Network/index.ts

```diff
@@ -6,14 +6,24 @@ import pkg from '../../../package.json';
 import * as MainQueue from './MainQueue';
 import * as SequentialQueue from './SequentialQueue';
 
+let processQueueInterval: NodeJS.Timer;
+
 // We must wait until the ActiveClientManager is ready so that we ensure only the "leader" tab processes any persisted requests
 ActiveClientManager.isReady().then(() => {
     SequentialQueue.flush();
 
     // Start main queue and process once every n ms delay
-    setInterval(MainQueue.process, CONST.NETWORK.PROCESS_REQUEST_DELAY_MS);
+    processQueueInterval = setInterval(MainQueue.process, CONST.NETWORK.PROCESS_REQUEST_DELAY_MS);
```

Comment:
I added some code to my testing branch, merged that in, and verified that we don't really need this. If we fail to fetch when re-authenticating, close the app, and re-open it later, then the app will immediately make a read request and trigger the re-authentication retry again. It will restart the process but that's fine. So all good on this one.

Here's proof of my test.

https://github.com/user-attachments/assets/e09dfd64-4bac-4cff-9060-605fa17530e0